src/EditorFeatures/CSharpTest/Diagnostics/GenerateMethod/GenerateMethodTests.cs

```diff
@@ -2532,18 +2532,237 @@ class Program
 {
     static void Main(string[] args)
     {
-        Func<int, int> f = x => Goo(x);
+        Func<string, int> f = x => Goo(x);
```

Comment:
these tests seem to have changed?  is thatbecause new similar tests were added? if not, can we restore the tests to the original state and add new tests?

Reply:
I changed it so that the return type is different than the parameter type. This would help catch bugs if they got mixed up during the generation.

Reply:
can you do that as new tests?

Reply:
If you don't like this change, I'd prefer to revert it instead. I don't think adding a new one adds any value...